components/progressbar/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spectrum-css/progressbar",
-  "version": "4.0.0",
+  "version": "5.0.0-next.0",
```

Comment:
I'm not in love with the way that the versioning is playing out on this `spectrum-two` branch while we try to keep pace with releases and versions on `main`, but...this is _probably_ our best bet.